tests/test_conversationhandler.py

```diff
@@ -277,3 +277,27 @@ def test_channel_message_without_chat(self, bot):
         message = Message(0, None, None, Chat(0, Chat.CHANNEL, 'Misses Test'), bot=bot)
         update = Update(0, message=message)
         assert not handler.check_update(update)
+
+    def test_conversation_timeout(self, dp, bot, user1):
```

Comment:
Related to the comment above, perhaps adding a test for 2 users would be prudent.